assets/components/headers/countrySwitcherHeader/countrySwitcherHeader.jsx

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import { SvgGuardianLogo } from 'components/svg/svg';
+import CountryGroupSwitcher from 'components/countryGroupSwitcher/countryGroupSwitcher';
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+
+type PropTypes = {
+  countryGroupIds: CountryGroupId[],
+  selectedCountryGroup: CountryGroupId,
+  onCountryGroupSelect: (string) => void,
+};
+
+
+// ----- Component ----- //
+
+const CountrySwitcherHeader = (props: PropTypes) => (
+  <header className="component-simple-header">
```

Comment:
Shouldn't this be `component-country-switcher-header`?

Reply:
Goo catch, fixed!